app/src/config/constants/sub/resoureTypeFilters.ts

```diff
@@ -0,0 +1,89 @@
+const RESOURCE_SUBSTR = {
+  html: "html",
+  json: "json",
+  js: ["javascript", "ecmascript"],
+  css: "css",
+  img: "image/",
+  media: ["video/", "audio/"],
+  form: "form",
+  font: "font",
+  misc: "", // none the above
+};
+
+interface FilterOption {
+  label: string;
+  value: keyof typeof RESOURCE_SUBSTR;
+}
+
+// because we wanted to control the order of filter options
+export const RESOURCE_FILTER_OPTIONS: FilterOption[] = [
+  {
+    label: "HTML Document",
```

Comment:
Let's simply call it `HTML`, as we don't know if HTML is fetched in AJAX or as document page in browser.